### PR TITLE
Migrate from npm to yarn

### PR DESCRIPTION
**Pull request by TheAznShumai**
Why:

* We need a better package and dependency management.

This change addresses the need by:

* Converting npm usage to yarn in package.json
* Updating Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The microsite for [Spartan's booster kits](http://boosters.joinspartan.com) was
 
 ## Requirements
 * node `^6.7.0`
-* npm `^3.10.6`
+* yarn `^0.19.1`
```
Comment (bbrock25) [inline README.md]: can we add a link to yarn installation instructions?

Comment (TheAznShumai) [inline README.md]: good idea

Comment (bbrock25) [inline README.md]: thanks dude.

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,5 @@ dist
 coverage
 
 .env
+
+yarn.lock
```
Comment (samcdavid) [inline .gitignore]: Shouldn't this be left in like `Gemfile.lock` on a Ruby project?

**Comment by gabeweaver** [comment]
@TheAznShumai thanks Johnny!!!!

**Pull request merged**